src/main/java/org/folio/tag/repository/TagRepositoryImpl.java

```diff
@@ -189,7 +190,7 @@ private String getTableName(String tenantId) {
   private CompletableFuture<Boolean> unAssignTags(AsyncResult<SQLConnection> connection, String tenantId, String recordId,
       RecordType recordType) {
     CompletableFuture<Boolean> future = new CompletableFuture<>();
-    final String deleteQuery = String.format(DELETE_TAG_RECORD, getTableName(tenantId), recordId, recordType);
```

Comment:
small fix here, 
DELETE_TAG_RECORD only has one "%s" placeholder,
recordId, recordType parameters are passed to postgresClient later